charts/manticoresearch/templates/manticore-balancer.yaml

```diff
@@ -25,6 +26,7 @@ spec:
       {{- end }}
       labels:
         name: {{ include "manticoresearch.fullname" . }}-balancer
+        release: {{ .Release.Name }}
```

Comment:
This label is also duplicated in `include "manticoresearch.labels"`. Remove it please

Reply:
Done